source/interaction/interaction_utilities.cc

```diff
@@ -1135,6 +1128,140 @@ namespace fdl
 #undef ARGUMENTS
   }
 
+  std_cxx17::optional<double>
+  intersect_line_with_edge(const std::array<Point<2>, 2> &simplex,
+                           const Point<2>                &stencil_start,
+                           const double                  &stencil_width,
+                           const unsigned int             stencil_axis)
+  {
+    dealii::Tensor<1, 2> q;
+    q[stencil_axis]           = 1;
+    const dealii::Point<2> p0 = simplex[0];
+    const dealii::Point<2> p1 = simplex[1];
+    double                 a, b;
```

Comment:
```suggestion
    Tensor<1, 2> q;
    q[stencil_axis]           = 1;
    const Point<2> p0 = simplex[0];
    const Point<2> p1 = simplex[1];
    double                 a, b;
```
In fiddle we always start with `using namespace dealii;` and `using namespace SAMRAI;` to cut down on typing.